src/main/scala/com/github/sbt/sbom/BomExtractor.scala

```diff
@@ -77,13 +98,16 @@ class BomExtractor(settings: BomExtractorParams, report: UpdateReport, log: Logg
       component.setPurl(
         new PackageURL(PackageURL.StandardTypes.MAVEN, group, name, version, new util.TreeMap(), null).canonicalize()
       )
+      if (settings.schemaVersion.getVersion >= Version.VERSION_11.getVersion) {
+        component.setBomRef(component.getPurl)
+      }
       component.setScope(Component.Scope.REQUIRED)
-      licenseChoice.foreach(component.setLicenseChoice)
+      component.setHashes(hashes(artifactPaths(moduleReport)).asJava)
```

Comment:
does this make things much more heavy? if so we might want to put this behind an option - but I'm fine with just having it always-included for now.